utils/dockerutil.py

```diff
@@ -392,6 +392,14 @@ def image_tag_extractor(cls, entity, key):
                     tags.add(split[key])
             if len(tags) > 0:
                 return list(tags)
+        elif entity.get('RepoDigests'):
+            # the human-readable tag is not mentioned in RepoDigests
+            if key > 0:
```

Comment:
so after that point if we didn't return, key is lower than 0 and we use it as an index. Seems wrong, no ?